leap.py

```diff
@@ -1,14 +1,13 @@
 # Function definition is here
 def is_leap_year( year ):
-   if year%4 == 0 and year%100 != 0:
-       print True
+   if (year%4 == 0 and year%100 != 0) or year%400 ==0:
+       print str(year) + " is a leap year"
+   else:
+        print str(year) + " is NOT a leap year"
    return;
```

Comment:
`return;` can be left out. All functions in Python return None (equivalent to `return` or `return None`) if no other return value specified.

As for `;` in `return;` - a semicolon after a statement is only useful in multi-statement lines. So such an example is correct Python (though not good style):
```
a = 1; b = 2
c = a + b
```
This technique is rarely useful as readability counts and cramming things into a single line makes it usually harder to read. I don't actually remember ever using it in a real code-base.

So it's better to leave it as:
```
a = 1
b = 1
c = a + b
```

See https://www.python.org/dev/peps/pep-0008/#other-recommendations ("Compound statements (multiple statements on the same line) are generally discouraged...").